treon/treon.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=line-too-long
```

Comment:
Should we disable this for entire repo?

Reply:
Actually, I think that Pylint has a good default of a maximum 100 characters per line, and longer lines are unreasonable (except for this one docstring).